docs/gensxg.md

```diff
@@ -39,23 +37,39 @@ $ gensxg -url https://example.com \
     > myFile.html.sxg
 ```
 
-## Options
+## DESCRIPTION
+
+A command-line tool for generating SXG (Signed HTTP Exchanges).
+
+## OPTIONS
+
+`help`:                 Show help message.
```

Comment:
Sorry, I think some dashes were removed erroneously.  I think the format should be:

```
`-help`:                 Show help message.
```

This will match `gencertchain.md` and show a single leading dash and no markdown bullet.

Reply:
sorry, it is mistake.